software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessWinRmDriver.java

```diff
@@ -154,6 +171,38 @@ public void runPostLaunchCommand() {
         }
     }
 
+
+    @Override
+    public void copyInstallResources() {
+        final WithMutexes mutexSupport = getLocation().mutexes();
+        String mutexId = "installation lock at host";
+        mutexSupport.acquireMutex(mutexId, "installing " + elvis(entity, this));
+        try {
+            super.copyInstallResources();
+        } catch (Exception e) {
+//            log.warn("Error copying install resources", e);
+            throw Exceptions.propagate(e);
+        } finally {
+            mutexSupport.releaseMutex(mutexId);
+        }
+    }
+
+    @Override
+    public void copyCustomizeResources() {
```

Comment:
What about `copyPreInstallResources`? Should we override that? No strong feelings whether we override `copyCustomizeResources` or not (we don't for ssh I think).